vulnerabilities/tests/test_data/github_api/npm-expected.json

```diff
@@ -0,0 +1,6175 @@
+          "version": null,
+          "qualifiers": null,
+          "subpath": null
+        },
+        "affected_version_range": "vers:npm/<8.8.1",
+        "fixed_version": null
+      }
+    ],
+    "references": [
+      {
+        "reference_id": "GHSA-98pf-gfh3-x3mp",
+        "url": "https://github.com/readthedocs/readthedocs.org/security/advisories/GHSA-98pf-gfh3-x3mp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "MODERATE",
+            "scoring_elements": ""
+          }
+        ]
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/readthedocs/readthedocs.org/commit/b0ae626acd13882170ec5888e35f3ef2e48e6ff6",
+        "severities": []
+      },
+      {
+        "reference_id": "GHSA-98pf-gfh3-x3mp",
+        "url": "https://github.com/advisories/GHSA-98pf-gfh3-x3mp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "MODERATE",
+            "scoring_elements": ""
+          }
+        ]
+      }
+    ],
+    "date_published": "2022-11-10T16:02:51+00:00"
+  },
+  {
+    "aliases": [
+      "GHSA-3qmc-2r76-4rqp"
+    ],
+    "summary": "Redwood is vulnerable to account takeover via dbAuth \"forgot-password\" ",
+    "affected_packages": [
+      {
+        "package": {
+          "type": "npm",
+          "namespace": null,
+          "name": "@redwoodjs/api",
+          "version": null,
+          "qualifiers": null,
+          "subpath": null
+        },
+        "affected_version_range": "vers:npm/>=3.0.0|<3.3.1",
+        "fixed_version": null
+      }
+    ],
+    "references": [
+      {
+        "reference_id": "GHSA-3qmc-2r76-4rqp",
+        "url": "https://github.com/redwoodjs/redwood/security/advisories/GHSA-3qmc-2r76-4rqp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "HIGH",
+            "scoring_elements": ""
+          }
+        ]
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/redwoodjs/redwood/issues/6343",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/redwoodjs/redwood/pull/6778",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/redwoodjs/redwood/releases/tag/v2.2.5",
+        "severities": []
+      },
+      {
+        "reference_id": "",
+        "url": "https://github.com/redwoodjs/redwood/releases/tag/v3.3.1",
+        "severities": []
+      },
+      {
+        "reference_id": "GHSA-3qmc-2r76-4rqp",
+        "url": "https://github.com/advisories/GHSA-3qmc-2r76-4rqp",
+        "severities": [
+          {
+            "system": "cvssv3.1_qr",
+            "value": "HIGH",
+            "scoring_elements": ""
+          }
+        ]
+      }
+    ],
+    "date_published": "2022-11-10T15:51:01+00:00"
+  },
+  {
+    "aliases": [
+      "GHSA-3qmc-2r76-4rqp"
```

Comment:
We seem to be missing the CVSS score and vector seen in https://github.com/redwoodjs/redwood/security/advisories/GHSA-3qmc-2r76-4rqp `CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:L`

Reply:
Tracked in https://github.com/nexB/vulnerablecode/issues/1045